activefires_pp/tests/test_geojson_utils.py

```diff
@@ -124,13 +124,18 @@ def test_get_recent_geojson_files(fake_past_detections_dir):
     pattern = "sos_{start_time:%Y%m%d_%H%M%S}_{id:d}.geojson"
 
     recent = get_recent_geojson_files(fake_past_detections_dir, pattern, (starttime, endtime))
-    assert recent == ['sos_20210618_124819_0.geojson', 'sos_20210619_000651_1.geojson', 'sos_20210619_000651_0.geojson']
+    assert len(recent) == 3
+    assert recent[0] == 'sos_20210618_124819_0.geojson'
+    assert set(recent[1:3]) == set(['sos_20210619_000651_1.geojson', 'sos_20210619_000651_0.geojson'])
```

Comment:
```suggestion
    assert set(recent) == {'sos_20210618_124819_0.geojson', 'sos_20210619_000651_1.geojson', 'sos_20210619_000651_0.geojson'}
```

Reply:
Off course, many thanks for spotting it.